apps/full-site-editing/full-site-editing-plugin/new-blocks-showcase/index.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * New Blocks Showcase
+ *
+ * @package A8C\FSE
+ */
+
+add_action(
+	'init',
+	function () {
+		$asset_file          = include plugin_dir_path( __FILE__ ) . 'dist/new-blocks-showcase.asset.php';
+		$script_dependencies = $asset_file['dependencies'];
+
+		wp_enqueue_script(
+			'new-blocks-showcase',
+			plugins_url( 'dist/new-blocks-showcase.js', __FILE__ ),
+			is_array( $script_dependencies ) ? $script_dependencies : array(),
+			filemtime( plugins_url( 'index.js', __FILE__ ) ),
+			false // We need this script before any blocks are registered.
```

Comment:
Enqueueing the script in the head seems to do the job, but I'm not sure if it actually *ensures* that it's run before any block is registered. Any thoughts on this, @marekhrabe?

Reply:
I don't have any better idea then this. It is likely skipping some (core) blocks but I think all other plugins enqueue way after `init` so we should be safe.